packages/xo-web/src/xo-app/proxies/index.js

```diff
@@ -159,12 +168,28 @@ export default decorate([
   addSubscriptions({
     proxies: subscribeProxies,
   }),
-  ({ proxies, router }) => (
+  provideState({
+    computed: {
+      licenses: () => getLicenses({ productType: 'xoproxy' }),
+      validLicensePerAppliance: ({ licenses = [] }) => {
```

Comment:
[[...]](https://github.com/vatesfr/xen-orchestra/pull/4916#discussion_r406277784)
But *proxy* appliance, right? I think "appliance" is confusing especially when used alone. We need to know if it's the XOA or the proxy just by reading the code.

Reply:
What about `validLicensePerProxyVm`?

Reply:
Yes.